scripts/index.js

```diff
@@ -0,0 +1,15 @@
+const menuIcon = document.querySelector('.mobile-menu-icon');
+const mobileNav = document.querySelector('.mobile-nav');
+const closeBtn = document.querySelector('.close-btn');
+
+function displayMenu() {
+  mobileNav.classList.add('show');
+  mobileNav.classList.remove('hide');
+}
+function closeMenu() {
+  mobileNav.classList.remove('show');
+  mobileNav.classList.add('hide');
+}
```

Comment:
- [ ] Your mobile menu does disappear when clicking the (X) icon but when clicking on any link it does not disappear as it is required. Please, kindly fix this issue in order to have a better user experience :smiley: :pushpin:
![image](https://user-images.githubusercontent.com/81522435/145845860-9fe3c7a0-06e0-44e4-b5ec-1fe473ff7027.png)